index.js

```diff
@@ -37,16 +38,38 @@ program
           options.model,
           Temperature
         );
-        return response.choices[0].message.content;
+
+        return {
+          content: response.choices[0].message.content,
+          tokensInfo: {
+            prompt: response.usage.prompt_tokens,
+            response: response.usage.completion_tokens
+          }
+        };
       }));
       
-      const output = responses.join('\n\n=================================================================================\n\n'); 
+      const output = responses.map(r => r.content).join('\n\n=================================================================================\n\n'); 

```

Comment:
I prefer variables a bit more meaningful rather than a single letter. Helps me when I'm looking back at this code in a week.

```suggestion
      const output = responses.map(response => response .content).join('\n\n=================================================================================\n\n'); 

```

Reply:
Fixed with https://github.com/aamfahim/explainer.js/pull/23/commits/7b0c2f2c471cb50bd15c4f8c1a2011b8bcf9e2e4